src/network.rs

```diff
@@ -113,6 +132,17 @@ impl Network {
         Ok(())
     }
 
+    async fn block_verify_tx_broadcaster(&self) -> Result<(), String> {
```

Comment:
maybe you can optimize this repeated code with `broadcast_message()`.

Reply:
resolved.